numpy/lib/tests/test_arraypad.py

```diff
@@ -12,6 +12,20 @@
 from numpy.lib.arraypad import _as_pairs
 
 
+_all_modes = {
```

Comment:
At some point it made sense to introduce this global variable to avoid duplication.